cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/preview/PreviewManagerTest.java

```diff
@@ -169,19 +169,6 @@ public void finish() {
     txManager.stopAndWait();
   }
 
-  @Test(expected = IllegalStateException.class)
-  public void testLimit() throws Exception {
```

Comment:
Removed this test case since the appInjector limit is removed now. Will add similar test case for preview job queue capacity.